baxter_description/launch/rviz_simulation.launch

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+
+<launch>
+  <!-- send robot XML to param server -->
+  <param name="robot_description" command="$(find xacro)/xacro --inorder $(find
```

Comment:
Hadn't thought of that - fixed: https://github.com/RethinkRobotics/baxter_common/pull/86/commits/43279691626128772a1f83317acd433f8f8ef264